test/unit/sthE2E_test.js

```diff
@@ -0,0 +1,142 @@
+                }
+            );
+        });
+
+        after(function(done) {
+            sth.sthServer.stopServer(function() {
+                done();
+            });
+        });
+
+        it('should return 500 when GET http request (e2e)', function(done) {
+            request(
+                {
+                    uri: sthTestUtils.getURL(
+                        sthTestConfig.API_OPERATION.READ,
+                        {
+                            lastN: 0,
+                            dateFrom:
+                                contextResponseNumericWithFixedTimeInstantV1.contextResponses[0].contextElement
+                                    .attributes[0].metadatas[0].value,
+                            dateTo:
+                                contextResponseNumericWithFixedTimeInstantV1.contextResponses[0].contextElement
+                                    .attributes[0].metadatas[0].value
+                        },
+                        contextResponseNumericWithFixedTimeInstantV1.contextResponses[0].contextElement.attributes[0]
+                            .name
+                    ),
+                    method: 'GET',
+                    headers: {
+                        Accept: 'application/json',
+                        'Content-Type': 'application/json',
+                        'Fiware-Service': sthConfig.DEFAULT_SERVICE,
+                        'Fiware-ServicePath': sthConfig.DEFAULT_SERVICE_PATH
+                    }
+                },
+                function(err, response) {
+                    expect(err).to.equal(null);
+                    expect(response.statusCode).to.equal(500);
+                    expect(JSON.parse(response.body).message).to.equal('DB not connected');
+                    done(err);
+                }
+            );
+        });
+
+        it('should return 500 when DELETE http request (e2e)', function(done) {
+            request(
+                {
+                    uri: sthTestUtils.getURL(
+                        sthTestConfig.API_OPERATION.DELETE,
+                        {
+                            entityType: 'T',
+                            entityId: 'E',
+                            attrName: 'A'
+                        },
+                        'A' // maybe unneeded in API_OPERATION.DELETE case (see getURL method)
+                    ),
+                    method: 'DELETE',
+                    headers: {
+                        Accept: 'application/json',
+                        'Content-Type': 'application/json',
+                        'Fiware-Service': sthConfig.DEFAULT_SERVICE,
+                        'Fiware-ServicePath': sthConfig.DEFAULT_SERVICE_PATH
+                    }
+                },
+                function(err, response) {
+                    expect(err).to.equal(null);
+                    expect(response.statusCode).to.equal(500);
+                    expect(JSON.parse(response.body).message).to.equal('DB not connected');
```

Comment:
Why don't cover all the pieces? :) I mean:

```
expect(JSON.parse(response.body).statusCode).to.equal(500);
expect(JSON.parse(response.body).error).to.equal('Internat Server Error');
expect(JSON.parse(response.body).message).to.equal('DB not connected');
```

(Also in the other test)

Reply:
Done in https://github.com/telefonicaid/fiware-sth-comet/pull/583/commits/e61b1cf33df00647a2f5b149bc08ff379f5a0dc2